test/ToolsTestFileIO.cpp

```diff
@@ -633,11 +633,11 @@ TEST_F(TestFileIO, TestSudoFileReadAsciiFileContent) {
    ASSERT_NE(targetGID, 0);
 
    //Open a common root permissioned file without root permissions.
-   auto badResult = FileIO::ReadAsciiFileContent("/sys/module/drm/parameters/debug");
+   std::string filePath("/sys/module/hid/parameters/debug");
```

Comment:
This would be more predictable if the test created a file with root permissions in /tmp/TestFileIO. Also these tests could check to see if the unit tests are being run as root or not and skip them if not root.